README.md

```diff
@@ -111,6 +111,11 @@ docker-compose -f docker-compose-demo.yml up
 
 You can also use this file as starting point for your production docker-compose setup.
 
+## See it in action
+
+You can check our latest release in Heroku by simply clicking the button below.
```

Comment:
Should we put some sort of disclaimer here that you need a Heroku account to do this?